### PR TITLE
Jakarta Annotation 2.0 milestone draft

### PR DESCRIPTION
**Pull request by m0mus**
Signed-off-by: Dmitry Kornilov <dmitry.kornilov@oracle.com>

## Specification PR template
When creating a specification project release review, create two PRs with the content divided as follows.

Include the following in PR#1:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of wombat-spec-x.y.pdf
- [x] Specification HTML in the form of wombat-spec-x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [ ] For a Progress Review, that sufficient progress has been made on
      a Compatible Implementation and TCK, to ensure that the spec is
      implementable and testable.
- [ ] For a Release Review, a summary that a Compatible Implementation
      is complete, passes the TCK, and that the TCK includes sufficient
      coverage of the specification. The TCK users guide MUST include
      the instructions to run the compatible implementations used to
      validate the release.  Instructions MAY be by reference.
- [x] The URL of the OSSRH staging repository for the api, javadoc:
      https://search.maven.org/artifact/jakarta.annotation/jakarta.annotation-api/2.0.0-RC1
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      http://download.eclipse.org/ee4j/jakartaee-tck/master/nightly/annotations-tck-2.0.0.zip
- [ ] The URL of the compatibility certification request issue:
      <add URL here>

Include the following in PR#2:
- [ ] Specification JavaDoc in the wombat/x.y/apidocs directory.

### CODE REVIEW

File: annotations/2.0/_index.md [hunk 1]
```diff
@@ -1,51 +1,23 @@
...
 * [Jakarta Annotations 2.0 TCK]()([sig](),[sha](),[pub]())
 * Maven coordinates
-  * [jakarta.annotation:jakarta.annotation-api:jar:2.0.0]()
-
+  * [jakarta.annotation:jakarta.annotation-api:jar:2.0.0-RC1](https://search.maven.org/artifact/jakarta.annotation/jakarta.annotation-api/2.0.0-RC1/jar)
 
 # Compatible Implementations
 
-* [Jakarta Annotations]()
+* [Jakarta Annotations](https://eclipse-ee4j.github.io/common-annotations-api/)
```
Comment (kwsutter) [inline annotations/2.0/_index.md:21]: This should be pointing at a specific download for a Compatible Impl.  Is there an RC for an Annotations CI?

File: annotations/2.0/_index.md [hunk 2]
```diff
@@ -1,51 +1,23 @@
...
 
 # Ballots
-
```
Comment (kwsutter) [inline annotations/2.0/_index.md:24]: Do not delete this material.  This is all good material for the Release Plan Review results.

File: annotations/2.0/_index.md (hunk 1 above)
Comment (m0mus) [reply annotations/2.0/_index.md:21]: There is no CI for annotations. Shall I put GlassFish there?

**Comment by m0mus** [comment]
@kwsutter I addressed all your comments. Please re-review.

**Pull request merged**